.github/workflows/push.yaml

```
@@ -74,7 +74,7 @@ jobs:
           variants=(
             'windows-x64;zip|exe'
             'macos-universal;tar.xz|pkg'
-            'ubuntu-22.04-x86_64;tar.xz|deb|ddeb'
+            'ubuntu-*-x86_64;tar.xz|deb|ddeb'
```

```suggestion
            'ubuntu-24.04-x86_64;tar.xz|deb|ddeb'
```
This should stay in sync with the value used for the artefact itself. Alternatively the distribution version could be removed from it as well, so that it's not even necessary here (just like with the Windows and macOS artefact).

The only reason it was added was to provide builds for 22.04 and 24.04 during a transitional period, so if that is still considered to be important in general, then the glob pattern should continue to use the explicit form.

---

Thanks for the feedback, sounds totally fine